ext/pdo_firebird/tests/CONFLICTS

```diff
@@ -0,0 +1 @@
+firebird
```

Comment:
Looks like unrelated commit snuck in?

Reply:
rebase issue... will be ignored on merge as already present (and I always only "git am" the relevant commit)